### PR TITLE
Added array_vacuum support

### PR DESCRIPTION
**Pull request by gsvic**

### CODE REVIEW

File: src/main/java/io/tiledb/java/api/Array.java [hunk 1]
```diff
@@ -370,6 +370,19 @@ public static void consolidate(
...
 
+  /**
+   * Cleans up the array, such as consolidated fragments and array metadata. Note that this will
+   * coarsen the granularity of time traveling (see docs for more information).
+   *
+   * @param arrayURI The array URI
+   * @param arrayURI The array URI
+   * @throws TileDBError A TileDB exception
+   */
+  public static void vacuum(Context ctx, String arrayURI) throws TileDBError {
```
Comment (Shelnutt2) [inline src/main/java/io/tiledb/java/api/Array.java:381]: Can we add an overload which takes a config object? The details should highlight that the config will be used for the consolidation parameters.

Add a note to this function that the contex's config will be used.

File: src/main/java/io/tiledb/java/api/Array.java [hunk 2]
```diff
@@ -370,6 +370,19 @@ public static void consolidate(
     }
   }
 
+  /**
+   * Cleans up the array, such as consolidated fragments and array metadata. Note that this will
+   * coarsen the granularity of time traveling (see docs for more information).
+   *
+   * @param arrayURI The array URI
```
Comment (Shelnutt2) [inline src/main/java/io/tiledb/java/api/Array.java:377]: Missing param description for Contex.

Comment (gsvic) [reply src/main/java/io/tiledb/java/api/Array.java:380]: done

File: src/main/java/io/tiledb/java/api/Array.java (hunk 1 above)
Comment (gsvic) [reply src/main/java/io/tiledb/java/api/Array.java:384]: done

**Pull request merged**